tests/backup/backup_helper.go

```diff
@@ -58,6 +62,11 @@ const (
 	mongodbPodStatusRetryTime                 = 30 * time.Second
 	backupLocationDeleteTimeout               = 30 * time.Minute
 	backupLocationDeleteRetryTime             = 30 * time.Second
+	latestPxBackupVersion                     = "2.4.0"
```

Comment:
Would it be a better approach if we could fetch this rom env?

Reply:
My opinion: if we want a particular commit to always upgrade only to a "specific latest version", then it makes sense to hardcode it (like a particular test being designed for particular version, I guess?). 
But if we want _any_ commit to always upgrade to the latest (or version to upgrade to) as defined by the user (env var), it yeah, env var is better.

Reply:
This is the just an indication of what is the latest version of Px Backup currently going on. User can pass any string for `versionToUpgrade`

Reply:
Cool